components/inputs/input-checkbox.js

```diff
@@ -62,6 +63,10 @@ class InputCheckbox extends RtlMixin(LitElement) {
 				.d2l-input-checkbox-text-disabled {
 					opacity: 0.5;
 				}
+				input.d2l-input-checkbox.align-top {
+					vertical-align: top;
+					margin-top: 0.2em;
+				}
```

Comment:
How about using an attribute selector instead? Should be able to accomplish the same effect with no JS changes.

```css
:host([align-top]) .d2l-input-checkbox {
    margin-top: 0.2em;
    vertical-align: top;
}
```